velox/exec/OperatorUtils.h

```diff
@@ -146,4 +146,13 @@ void projectChildren(
     int32_t size,
     const BufferPtr& mapping);
 
+/// Overload of the above function that takes reference to const vector of
+/// VectorPtr as 'src' argument, instead of row vector.
+void projectChildren(
```

Comment:
This function is not used yet, right? Are you using it in your other PR?

It would be nice to update this PR's description to clarify that you add a new helper function.

Reply:
It's used at https://github.com/facebookincubator/velox/blob/3f8144b62c95e2c4ae7f41c958ccdf020694ecf2/velox/exec/Operator.cpp#L205-L210

Since `results_` is `vector<VectorPtr>`.